.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Link Checker
+      id: lc
+      uses: peter-evans/link-checker@v1
+      with:
+        args: -v -r docs
```

there are links elsewhere in the repo (i.e. in the website/ folder), how does it perform if we changed it to this instead?
```suggestion
        args: -v -r *
```